src/utils/strings.js

```diff
@@ -58,3 +58,21 @@ export function findUrls(str) {
     }
   })
 }
+
+/**
+ * Convert string to finite float.
+ *
+ * @param {string} str Input string.
+ * @return {number} float.
+ */
+export function stof(str) {
+  const floatDigits = 2
```

Comment:
pls move this up to a param `len=3`; I added a comment asking for the default to be changed from 2 to 3:

https://github.com/bldrs-ai/Share/issues/544